Example/InstanceID/Tests/FIRInstanceIDTest.m

```diff
@@ -988,6 +989,21 @@ - (void)testInstanceIDDelete_keyChainError {
   [self waitForExpectationsWithTimeout:1 handler:nil];
 }
 
+- (void)testInstanceIDConfigureWithOptionsSetsUserAgentForTokenManager {
+  // The shared instance relies on the default app being configured. Configure it.
+  FIROptions *options = [[FIROptions alloc] initWithGoogleAppID:kGoogleAppID
+                                                    GCMSenderID:kGCMSenderID];
+
+  FIRApp *app = [[FIRApp alloc] initInstanceWithName:@"FIRInstanceIDTest" options:options];
+
+  NSString *expectedUserAgent = [[app class] firebaseUserAgent];
+  OCMExpect([self.mockTokenManager setFirebaseUserAgent:[OCMArg isEqual:expectedUserAgent]]);
```

Comment:
I think you don't need to use `OCMArg` here and can directly use:

```
OCMExpect([self.mockTokenManager setFirebaseUserAgent:expectedUserAgent]);
```

I could be wrong though :)

Reply:
You are right! I'll update all `[OCMArg isEqual:]` I introduced. Thanks.

Reply:
Though, I am just thinking, if it would be a bit more readable to keep `[OCMArg isEqual:]`? Then it will be obvious from the first glance that we are not passing anything to the method but just validating the input parameters.
@ryanwilson What do you think?

Reply:
Sounds reasonable to me as long as the validation is being done 👍

Reply:
Then I'll keep `[OCMArg isEqual:]`